postgres/lsmb-dev-init.sh

```diff
@@ -1,7 +1,7 @@
 #!/bin/bash
 
 
-pgdata=${PGDATA:-/var/lib/postgresql/data}
+pgdata=${PGDATA:-/var/lib/postgresql/data/pgdata}
```

Comment:
Why this change? PostgreSQL explicitly documents that it's data directory should *not* be a mountpoint which is why we have this directory in place the way it is now.

Or are you saying that the bit below doesn't work? Then why does my container log all queries?

Reply:
postgresql.conf is in `/var/lib/postgresql/data/pgdata`, not `/var/lib/postgresql/data`

Reply:
Yours work because the docker.compose.yml file sets PGDATA properly

Reply:
Ah! Thanks for that clarification.